components/navbar.vue

```diff
@@ -46,8 +79,15 @@
 import { defineComponent } from '@nuxtjs/composition-api'
 
 export default defineComponent({
+  data() {
+    return {
+      initials: '',
+      name: '',
+    }
+  },
   computed: {
     auth() {
+      this.icon()
```

Comment:
you should have used watch on auth and then should have called icon rather than this